metrics/windows/disk.go

```diff
@@ -106,7 +106,7 @@ func (g *DiskGenerator) Generate() (metrics.Values, error) {
 	results := make(map[string]float64)
 	for _, v := range g.counters {
 		var value windows.PDH_FMT_COUNTERVALUE_ITEM_DOUBLE
-		r, _, err := windows.PdhGetFormattedCounterValue.Call(uintptr(v.Counter), windows.PDH_FMT_DOUBLE, uintptr(0), uintptr(unsafe.Pointer(&value)))
+		r, _, err := windows.PdhGetFormattedCounterValue.Call(uintptr(v.Counter), windows.PDH_FMT_DOUBLE, uintptr(0), uintptr(unsafe.Pointer(&value.FmtValue)))
```

Comment:
It looks like `PDH_FMT_COUNTERVALUE_ITEM_DOUBLE` is not used.
I think it is better to use `var value PDH_FMT_COUNTERVALUE_DOUBLE` and to get rid of all `PDH_FMT_COUNTERVALUE_ITEM_DOUBLE`.